### PR TITLE
UIDS-4 reorganize scss files under src

### LINKED ISSUE #4: Reorganize styling file structure

### PR DESCRIPTION
**Pull request by davidmferris**
Closes #4 

Per offline conversation, we're going to keep the scss directory at root level. I added component level directories to hold the components and their styles (see the Pill component for an example).

### CODE REVIEW

File: stories/Card.stories.jsx [hunk 1]
```diff
@@ -2,13 +2,13 @@ import React from 'react';
...
 
-import Card from '../src/Card';
+import Card from '../src/Card/Card';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
 import '../scss/global.scss';
 
 export default {
-  title: 'Design System|Card',
+  title: 'Design System/Card',
```
Comment (davidmferris) [inline stories/Card.stories.jsx]: These symbols handle story hierarchy in the storybook explorer. `|` is being deprecated in favor of `/`.

File: src/Pill/Pill.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@import '../../scss/theme';
+
+.pill {
```
Comment (rsaris) [inline src/Pill/Pill.scss]: Should this be `Pill`?

File: src/Pill/Pill.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import './Pill.scss';
+
+const Pill = ({ color, size, text }) => (
+  <div
+    className={
+      classNames(
+        'pill',
```
Comment (rsaris) [inline src/Pill/Pill.jsx]: With below, should we start using the `Pill` casing for these classes?

Comment (davidmferris) [reply src/Pill/Pill.jsx]: Sounds reasonable. Will update.

Comment (davidmferris) [reply src/Pill/Pill.jsx]: Sounds reasonable. Will update.

**Pull request merged**